documentation/01-Prerequisites.md

```diff
@@ -6,76 +6,46 @@ These are one-time setup instructions that should be executed prior to following
 
 - [Prerequisites for building and using MLOS](#prerequisites-for-building-and-using-mlos)
   - [Contents](#contents)
-  - [Requirements](#requirements)
-  - [Clone the repository](#clone-the-repository)
-  - [Install build tools](#install-build-tools)
-    - [Docker](#docker)
+  
+  - [Linux](#linux)
+    - [Requirements](#linux-requirements)
+    - [Clone the repository](#clone-the-repository)
     - [Linux build tools](#linux-build-tools)
+      - [Linux Build Manually](#linux-build-manually)
+      - [Docker](#docker)
+    - [Linux Python Install](#linux-python-install)
+    - [Linux Docker Install](#linux-docker-install)
+  
+  - [Windows](#Windows)
+    - [Windows Requirements](#windows-requirements)
+    - [Clone the repository](#clone-the-repository)
     - [Windows build tools](#windows-build-tools)
       - [Using a local script](#using-a-local-script)
       - [Using Chocolatey](#using-chocolatey)
-      - [Manually](#manually)
-  - [Install Python Dependencies](#install-python-dependencies)
-    - [Linux](#linux-python-install)
-    - [Windows](#windows-python-install)
-  - [Install Docker](#install-docker)
-    - [Linux Docker Install](#linux-docker-install)
+      - [Windows Build Manually](#win-build-manually)
+    - [Windows Pythoon Install](#windows-python-install)
     - [Windows Docker Install](#windows-docker-install)

```

Comment:
Should see about having vscode automatically update this for us.  Might need an extension.  Perhaps one of these?
davidanson.vscode-markdownlint
shd101wyy.markdown-preview-enhanced

Reply:
Nope it was joffreykern.markdown-toc or yzhang.markdown-all-in-one